src/Donations/Components/PaymentsForm.tsx

```diff
@@ -120,7 +120,7 @@ function PaymentsForm({}: Props): ReactElement {
     const gateway = "stripe";
     onSubmitPayment(
       gateway,
-      "card",
+      paymentRequest._activeBackingLibraryName,
```

Comment:
What's the purpose if this? This parameter is used as `method` in  `onSubmitPayment()` and directly used for calling https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a4ba73ca01da3c21d9088aeb4ba9317d69a35864/src/Donations/Components/PaymentsForm.tsx#L98-L100 which again is giving this method to `buildPaymentProviderRequest()` https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a4ba73ca01da3c21d9088aeb4ba9317d69a35864/src/Donations/PaymentMethods/PaymentFunctions.ts#L238-L243 and `handleStripeSCAPayment()` https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a4ba73ca01da3c21d9088aeb4ba9317d69a35864/src/Donations/PaymentMethods/PaymentFunctions.ts#L276-L278 who can not deal with other methods as `card` yet.

Reply:
Btw. I saw `handleStripeSCAPayment()` is called with a first parameter `gateway` which is not handled in it's definition at https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a4ba73ca01da3c21d9088aeb4ba9317d69a35864/src/Donations/PaymentMethods/PaymentFunctions.ts#L378-L380 - this is a bug.